docs/source/new-project.rst

```diff
@@ -0,0 +1,70 @@
+Quick Start
+===========
+To start a new project define two files, a LightningModule and a Trainer file.    
+To illustrate Lightning power and simplicity, here's an example of a typical research flow.    
```

Comment:
`to illustrate Lightning's`

Reply:
well there is not much new text (almost none)